frontend/packages/dev-console/src/components/import/image-search/ImageStream.tsx

```diff
@@ -18,7 +18,7 @@ import {
   getImageStreamTags,
 } from '../../../utils/imagestream-utils';
 import './ImageStream.scss';
-import ResourceDropdownField from '../../formik-fields/ResourceDropdownField';
+import ResourceDropdownField from '../../../../../console-shared/src/components/formik-fields/ResourceDropdownField';
```

Comment:
```suggestion
import { ResourceDropdownField } from '@console/shared';
```
After the above change is done.

Reply:
Done